src/views/studio/CreateEditChannelView/CreateEditChannelView.tsx

```diff
@@ -319,21 +313,29 @@ export const CreateEditChannelView: React.FC<CreateEditChannelViewProps> = ({ ne
       }
     }
 
-    const completed = await handleTransaction({
-      preProcess: processAssets,
-      txFactory: (updateStatus) =>
-        newChannel
-          ? joystream.createChannel(activeMemberId, metadata, assets, updateStatus)
-          : joystream.updateChannel(activeChannelId ?? '', activeMemberId, metadata, assets, updateStatus),
-      onTxFinalize: uploadAssets,
-      onTxSync: refetchDataAndCacheAssets,
-      successMessage: {
-        title: newChannel ? 'Channel successfully created!' : 'Channel successfully updated!',
-        description: newChannel
-          ? 'Your channel was created and saved on the blockchain. Feel free to start using it!'
-          : 'Changes to your channel were saved on the blockchain.',
-      },
-    })
+    const completed =
+      joystream &&
+      (await handleTransaction({
+        preProcess: processAssets,
+        txFactory: async (updateStatus) =>
+          newChannel
+            ? await joystream.createChannel(activeMemberId, metadata, assets, proxyCallback(updateStatus))
+            : await joystream.updateChannel(
+                activeChannelId ?? '',
+                activeMemberId,
+                metadata,
+                assets,
+                proxyCallback(updateStatus)
+              ),
+        onTxFinalize: await uploadAssets,
+        onTxSync: await refetchDataAndCacheAssets,
```

Comment:
Why do we have all these additional `await`s here? I don't think they add anything